src/ui/public/notify/toasts/TOAST_NOTIFICATIONS.md

```diff
@@ -0,0 +1,79 @@
+# Toast notifications
```

Comment:
See https://github.com/cjcenizal/kibana/blob/euify/notification-toasts-extract-fatal/src/ui/public/notify/toasts/TOAST_NOTIFICATIONS.md to preview this file in a more readable form.